lib/jrjackson/build_info.rb

```diff
@@ -1,27 +1,27 @@
 module JrJackson
   module BuildInfo
     def self.version
-      '0.4.16'
+      '0.4.17'
     end
 
     def self.release_date
-      '2022-07-08'
+      '2023-01-14'
     end
 
     def self.files
       git_files.concat(generated_jar_files).concat(generated_files)
     end
 
     def self.jackson_version
-      '2.13.3'
+      '2.14.0'
```

Comment:
2.14.0 (2.14.1 is out,btw) has pretty sizeable changeset and may require a bit more investigation to see if nothing breaks, were you able to run all the tests in jrjackson with different JVMs ?
As an alternative I suggest we update to [2.13.4.2](https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind/2.13.4.2) instead.

Reply:
I would suggest we get the latest version i.e. 2.14.1 and do bit more investigation, this would just full-proof and avoid orgs getting vulnerability tickets as 2.13.4.2 will have vulnerable cve's

Reply:
I have build `jrjackson` with `jackson-2.14.1` version and tested against Logstash (with unit test cases).